library.js

```diff
@@ -21,7 +21,9 @@
 		}
 	});
 
-	var Twitter = {};
```

Comment:
Please fix the indentation issues in this PR, as we use tabs, not spaces. Thanks.